app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,9 @@
 	<uses-permission android:name="android.permission.INTERNET" />
 	<uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
 	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	<permission android:name="org.openobservatory.ooniprobe.permission.C2D_MESSAGE" android:protectionLevel="signature" />
+	<uses-permission android:name="org.openobservatory.ooniprobe.permission.C2D_MESSAGE" />
```

Comment:
Can you explain to me what these two new permissions do? Any extra permission we could maybe remove?

Reply:
from the countly doc

Reply:
"Add extra lines in AndroidManifest.xml"
https://support.count.ly/hc/en-us/articles/360038842151-Android-SDK-19-02-

Reply:
This permission allows the cloud to send messages to the device.